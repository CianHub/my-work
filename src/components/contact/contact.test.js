// contact.contact.test.js
import React from "react";
import Contactmain from "./contact";
import renderer from "react-test-renderer";

test("Contact section is rendered", () => {
  const component = renderer.create(
    <Contactmain
      email="mailto:cian@cianogralaigh.com"
      linkedin="https://www.linkedin.com/in/cian-o-gralaigh/"
      github="https://github.com/CianHub"
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
