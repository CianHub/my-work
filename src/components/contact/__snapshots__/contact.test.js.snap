// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact section is rendered 1`] = `
<section
  id="contact"
>
  <div
    className="react-reveal"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <div
      className="container header"
    >
      <div
        className="row"
      >
        <div
          className="col-lg-12 text-center"
        >
          <h2
            className="section-heading"
            id="contactText"
          >
            
          </h2>
          <hr
            className="my-4"
            id="contactHr"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="react-reveal container "
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <div
      className="row text-center"
    >
      <div
        className="col-md"
      >
        <ul
          className="horizontal-list social-circle"
        >
          <li>
            <a
              className="email"
              href="mailto:cian@cianogralaigh.com"
              title="email"
            />
          </li>
          <li>
            <a
              className="linkedin"
              href="https://www.linkedin.com/in/cian-o-gralaigh/"
              title="linkedin"
            />
          </li>
          <li>
            <a
              className="github"
              href="https://github.com/CianHub"
              title="Github"
            />
          </li>
        </ul>
      </div>
    </div>
  </div>
  <p
    className="react-reveal text-muted text-center"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    Cian O'GralaighÂ©2018 
    <br />
    <br />
    <a
      className="back"
      href="#jumbo"
    >
      Back To Top
    </a>
  </p>
</section>
`;
